hr_calendar_zone/models/resource_calendar.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2016 Esther Martín - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class ResourceCalendarAttendance(models.Model):
+    _inherit = 'resource.calendar.attendance'
+
+    zone_id = fields.Many2one(
+        comodel_name='res.partner.zone', string='Zone')
+    employee_id = fields.Many2one(
+        comodel_name='hr.employee', string='Employee')
+
+    @api.onchange('employee_id')
+    def onchange_employee_id(self):
+        if self.employee_id:
```

Comment:
que pasa si el employee no tiene address_home_id.... y si ese partner no tiene zone_ids??
`[('id', 'in', [])]` permite seleccionar alguna zona? debe de permitir o no?